students/eowyn/session09/oo-mailroom/test_oomailroom.py

```diff
@@ -0,0 +1,148 @@
+#!/usr/bin/env python
+
+import oomailroom as mr
+import os.path
+
+
+def test_donor_init():
+    name = "Ada Lovelace"
+    amt = 345
+    d = mr.Donor(name)
+    d.add_donation(amt)
+    assert d.name == name
+    assert d.donations == [345]
+
+def test_name():
+    name = "Ada Lovelace"
+    amt = 345
+    d = mr.Donor(name)
+    d.add_donation(amt)
+    assert d.name == name
+
+
+def test_create_transactions():
+    name = "Ada Lovelace"
+    amt = 345
+    d = mr.Donor(name)
+    d.add_donation(amt)
+    t = mr.Transactions()
+    t.add_donor(name, amt)
```

Comment:
nothing wrong with the test, but this shows a kinda odd API.

you probably want to add the donation amount to  the donor first (or after) and then add the donor to the Transactions object. Adding a donor and transaction at the same time is not really separating your concerns.